tutorial/Lab1/README.md

```diff
@@ -0,0 +1,168 @@
+# Lab 1. I just want to deploy!
+
+We developers are a lazy bunch! If we can get away with just typing a single command over multiple, not worrying about how things tie together, then we are more than happy. Lets investigate how Helm can help with our lack of work ethic by first deploying an application to a Kubernetes cluster using `kubectl` and afterwards using Helm.
+
+The application is the [Guestbook App](https://github.com/IBM/guestbook) which is a a sample multi-tier web application.
+
+# Deploy the application using `kubectl`
+
+In this part of the lab we will deploy the application using the Kubernetes client `kubectl`. We will use [Version 1](https://github.com/IBM/guestbook/tree/master/v1) of the app for deploying here. Clone the [Guestbook App](https://github.com/IBM/guestbook) repo to get the files: 
+```$ git clone https://github.com/IBM/guestbook.git``` .
+
+1. Use the configuration files in the cloned Git repository to deploy the containers and create services for them, with the following commands:
```

Comment:
Remove the comma after `them`
Instead of `with the following commands` use `by using the following commands`

Reply:
Done